README.md

```diff
@@ -173,3 +173,124 @@ to determine that should this request be counted into rate-limit
 or not based on information given by `Request` object
 (i.e., Skip counting rate-limit on some route) and the key of the given request,
 by default, this will always return `false` which means counted everything.
+
+### getServer
+
+`(app: Elysia) => MaybePromise<Elysia | NeedRequestIP>`
+
+Default: `(app) => app`
+
+```ts
+import { Elysia } from 'elysia'
+import { rateLimit } from 'elysia-rate-limit'
+import bearer from '@elysiajs/bearer'
+
+class appInstance {
+  static server: Server | null
+}
+
+const setup = new Elysia({
+  name: 'setup',
+}).use(bearer())
+
+const userList = new Elysia({
+  name: 'userListRoute',
+})
+  .use(setup)
+  .use(
+    rateLimit({
+      getServer: _ => appInstance,
+      scoping: 'scoped',
+      max: 10,
+      duration: 100,
+    })
+  )
+  .get('/user', ({ bearer }) => {
+    return user.list()
+  })
+
+const userAdd = new Elysia({
+  name: 'userAddRoute',
+})
+  .use(setup)
+  .use(
+    rateLimit({
+      getServer: _ => appInstance,
+      scoping: 'scoped',
+      max: 2,
+      duration: 1000,
+    })
+  )
+  .post('/user', ({ body }) => {
+    return user.add(body)
+  })
+
+const app = new Elysia({
+  name: 'app',
+})
+  .use(helmet())
+
+  .use(userAdd)
+  .use(userList)
+
+  .listen(
+    {
+      port: 8081,
+    },
+    ({ development, hostname, port }) => {
+      console.log(
+        `🦊 Elysia is running at ${hostname}:${port} ${
+          development ? '🚧 in development mode!🚧' : ''
+        }`
+      )
+    }
+  )
+
+appInstance.server = app.server
+```
```

Comment:
example on documentation is too long, needed to simplify

Reply:
Please help.
I don't know how to simplify it.🥲
Because i think this only way to understand how `getServer` work.